Gemfile

```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
-ruby '2.4.4'
+ruby '2.5.1'
```

Comment:
Could you revert those changes to the version of Ruby, please? They don't make sense as part of this PR, and they seem to be breaking the build anyway.